ext/pcntl/pcntl.c

```diff
@@ -1385,6 +1386,11 @@ void pcntl_signal_dispatch()
 	sigprocmask(SIG_SETMASK, &old_mask, NULL);
 }
 
+static void pcntl_signal_dispatch_tick_function(int dummy_int, ZEND_ATTRIBUTE_UNUSED void *dummy_pointer)
```

Comment:
Why is one of these marked as unused and the other isn't? As we disable this warning, I'd not add the attribute.

Reply:
Because somehow I thought this function used the first param, clearly being drunk. Will remove the attribute.